File: server/src/main/java/com/noahgeren/trailangel/domain/EmergencyContact.java
package com.noahgeren.trailangel.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.fasterxml.jackson.annotation.JsonIgnore;

import lombok.Data;

@Data
@Entity
public class EmergencyContact {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	
	private String name;
	
	private String phoneNumber;
	
	private String user;
	
}

File: client/app/src/main/java/com/noahgeren/trailangel/models/EmergencyContact.kt
package com.noahgeren.trailangel.models

import androidx.room.Entity
import androidx.room.PrimaryKey
import java.io.Serializable

@Entity
data class EmergencyContact(
    @PrimaryKey var id: Long? = null,
    var name: String? = null,
    var phoneNumber: String? = null,
    var user: String? = null
): Serializable
